core/src/HIP/Kokkos_HIP_Graph_Impl.hpp

```diff
@@ -48,26 +48,26 @@ class GraphImpl<Kokkos::HIP> {
   GraphImpl& operator=(GraphImpl const&) = delete;
   GraphImpl& operator=(GraphImpl&&) = delete;
 
-  ~GraphImpl();
+  inline ~GraphImpl();
 
-  explicit GraphImpl(Kokkos::HIP instance);
+  explicit inline GraphImpl(Kokkos::HIP instance);
 
-  void add_node(std::shared_ptr<aggregate_node_impl_t> const& arg_node_ptr);
+  inline void add_node(std::shared_ptr<aggregate_node_impl_t> const& arg_node_ptr);
```

Comment:
I would prefer if you added `inline` to the definition instead of the declaration since the former is what causes the multiple defintions.

Reply:
Indeed, done!